migrations/2024011018230000-create-user-feedback.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.query(
+      `CREATE TABLE user_feedback (
+      id bigint NOT NULL AUTO_INCREMENT,
+      question_1 text,
+      question_2 text,
+      question_3 text,
+      question_4 text,
```

Comment:
For these question columns, I think we should choose more descriptive names, since we may want to add, remove, or change the order of questions in the future. Looking at the questions, how about something like:
1. question_use_case
2. question_impact
3. question_who_benefits
4. question_improvements

Reply:
Good call updated column names and test locally, all good.